pkgs/development/ocaml-modules/camlimages/4.1.2.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitLab, fetchpatch, omake, ocaml, findlib
+, graphicsmagick, ghostscript
+, freetype, giflib, libjpeg, libpng, libtiff, libXpm
+}:
+
+stdenv.mkDerivation rec {
+  pname = "camlimages";
+  version = "4.1.2";
+
+  src = fetchFromGitLab {
+    owner = "camlspotter";
+    repo = "camlimages";
+    rev = "98661d507e12ce91a51295a50f244cb8265b4439"; # no tag
+    sha256 = "0kpxj8wm2m17wjq217jzjpfgv1d7sp4w1yd1gi8ipn5rj4sid2j8";
+  };
+
+  buildInputs = [ ocaml omake findlib graphicsmagick ghostscript ];
+  propagatedBuildInputs = [ libtiff libjpeg libpng giflib freetype libXpm ];
+
+  createFindlibDestdir = true;
+
+  buildPhase = ''
+    omake
+  '';
+
+  installPhase = ''
+    omake install
+  '';
+
+  meta = with lib; {
+    branch = "4.0";
```

Comment:
Is this actually used somewhere?

Reply:
Yes, it's a dependency of glsurf, as a simple grep would have told you.

Reply:
glsurf is not using meta.branch anywhere.

Reply:
Oh, I'm sorry.  `branch` is a standard meta attribute.  Quoting the [manual](https://nixos.org/manual/nixpkgs/unstable/#var-meta-branch):

> Used to specify that a package is not going to receive updates that are not in this branch; for example, Linux kernel 3.0 is supposed to be updated to 3.0.X, not 3.1.

But I should have changed it to "4.1.2", since that's the last version we can use with OCaml 4.01.